util/oteltest/src/oteltest/private.py

```diff
@@ -0,0 +1,251 @@
+import glob
```

Comment:
Should we have the license at the top of all the privates, test files, and example files too? I'm not sure what's best since the examples in the core repo are inconsistent, e.g. [auto-instrumentation](https://github.com/open-telemetry/opentelemetry-python/blob/main/docs/examples/auto-instrumentation/client.py) vs [logs](https://github.com/open-telemetry/opentelemetry-python/blob/main/docs/examples/logs/example.py) examples. So maybe either way is fine!

Reply:
Added, thanks.